src/fe/fe_interface.C

```diff
@@ -545,9 +543,9 @@ FEInterface::n_dofs(const FEType & fe_t,
 
 #ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
 
-  // FIXME: Can/should InfElems account for p_level()?
+  // InfElems currently don't support p_level()
   if (is_InfFE_elem(elem->type()))
-    return ifem_n_dofs(dim, fe_t, elem->type());
+    return ifem_n_dofs(fe_t, elem);
```

Comment:
Since mesh refinement currently is not working with infinite elements at all (#1149), one probably doesn't have to care about this here?
Also, there are two separate polynomial orders: the ``base`` order and ``radial`` order; an implementation of p-refinement thus might take care of this?

Reply:
Yeah, this PR is mainly about having a consistent `FEInterface::n_dofs()` API, so the fact that InfFEs don't really support p-refinement at the moment isn't critical.